config/config.go

```diff
@@ -74,10 +68,17 @@ type EndPointConfig struct {
 
 // findCfg searches for the config file in this directory and all parents up the tree
 // looking for the closest match
-func findCfg() (string, error) {
-	dir, err := os.Getwd()
+func findCfg(path string) (string, error) {
+	var err error
+	var dir string
+	if path == "." {
+		dir, err = os.Getwd()
+	} else {
+		dir = path
+		_, err = os.Stat(dir)
+	}
```

Comment:
It will be tough if the case of empty characters is not taken into account.

Reply:
urfave will not allow to pass empty characters ""
argument for --configdir is required

Reply:
@mobiletoly 

I would like to make it optional, as I feel it is not backward compatible.